packages/patternfly-react/src/components/VerticalNav/__snapshots__/VerticalNav.test.js.snap

```diff
@@ -8,36 +8,157 @@ exports[`VerticalNav renders properly in mobile mode 1`] = `
     class="navbar-header"
   >
     <button
-      class="navbar-toggle collapsed"
-      type="button"
+      class="navbar-toggle"
```

Comment:
This snapshot file is not really indicating what changed as part of this PR.  It might be better to scope the snapshots using selectors or use targeted assertions for the easy stuff.